crates/gobley-uniffi-bindgen/src/main.rs

```diff
@@ -53,6 +53,10 @@ struct Cli {
     #[clap(long = "format", default_value_t = false)]
     try_format_code: bool,
 
+    /// Generate Kotlin Multiplatform bindings with all targets (jvm, android, native).
+    #[clap(long = "kmp")]
+    kotlin_multiplatform: bool,
```

Comment:
I think making `kotlin_multiplatform = true` by default is more reasonable, as the reason for preferring Gobley over Mozilla Kotlin bindgen would be mostly because of Kotlin Multiplatform support.